docker-image-src/4.0/docker-entrypoint.sh

```diff
@@ -206,6 +206,34 @@ function install_neo4j_labs_plugins
   rm "${_old_config}"
 }
 
+function process_ssl_policies
+{
+    local _found_a_policy_scope=false
+    local _ssl_policy_scopes=("bolt" "https" "cluster" "backup" "fabric")
+    for scope in ${_ssl_policy_scopes[@]}
+    do
+        echo $scope
```

Comment:
This echo should either be more explicit "checking if ssl policy for \"$scope\" is provided" or it should just be removed